addons/gdUnit4/src/core/command/GdUnitCommandHandler.gd

```diff
@@ -273,8 +273,19 @@ func cmd_create_test() -> void:
 func cmd_discover_tests() -> void:
 	await GdUnitTestDiscoverer.run()
 
-
-static func scan_test_directorys(base_directory: String, test_directory: String, test_suite_paths: PackedStringArray) -> PackedStringArray:
+static func scan_all_test_directories(override_root: String = "", force_override: bool = false) -> PackedStringArray:
+	var root: String
+	if override_root == "" and not force_override:
+		root = GdUnitSettings.test_root_folder()
+	else:
+		root = override_root
+	var base_directory := "res://"
```

Comment:
```suggestion
static func scan_all_test_directories(root: String) -> PackedStringArray:
	var base_directory := "res://"
```

Reply:
Place do not use default parameters, this could be used in a wrong way.
The caller should use the configured root path as argument.

Reply:
Will do!